### PR TITLE
Add missing error messages to 'ecma/builtin-objects'

### PR DESCRIPTION
**Pull request by LaszloLango**
Related issue: #897

JerryScript-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

File: jerry-core/ecma/builtin-objects/ecma-builtin-function-prototype.c [hunk 1]
```diff
@@ -387,7 +387,7 @@ ecma_builtin_function_prototype_dispatch_construct (const ecma_value_t *argument
 {
   JERRY_ASSERT (arguments_list_len == 0 || arguments_list_p != NULL);
 
-  return ecma_raise_type_error (ECMA_ERR_MSG (""));
+  return ecma_raise_type_error (ECMA_ERR_MSG ("'Function.prototype'' is not a constructor."));
```
Comment (zherczeg) [inline jerry-core/ecma/builtin-objects/ecma-builtin-function-prototype.c]: Double quote

File: jerry-core/ecma/builtin-objects/ecma-builtin-global.c [hunk 2]
```diff
@@ -774,7 +774,7 @@ ecma_builtin_global_object_decode_uri_helper (ecma_value_t uri, /**< uri argumen
 
     if (!lit_read_code_unit_from_hex (input_char_p + 1, 2, &decoded_byte))
     {
-      ret_value = ecma_raise_uri_error (ECMA_ERR_MSG (""));
+      ret_value = ecma_raise_uri_error (ECMA_ERR_MSG ("Invalid hexa value."));
```
Comment (zherczeg) [inline jerry-core/ecma/builtin-objects/ecma-builtin-global.c]: Hexadecimal.

File: jerry-core/ecma/builtin-objects/ecma-builtin-global.c [hunk 3]
```diff
@@ -1042,7 +1042,7 @@ ecma_builtin_global_object_encode_uri_helper (ecma_value_t uri, /**< uri argumen
 
     if (lit_is_code_point_utf16_low_surrogate (ch))
     {
-      ret_value = ecma_raise_uri_error (ECMA_ERR_MSG (""));
+      ret_value = ecma_raise_uri_error (ECMA_ERR_MSG ("Stray surrogate."));
```
Comment (zherczeg) [inline jerry-core/ecma/builtin-objects/ecma-builtin-global.c]: Unicode surrogate pair missing.

**Comment by LaszloLango** [comment]
@zherczeg, I've updated the PR. Please check.

**Comment by zherczeg** [comment]
LGTM

**Pull request merged**